### ISSUE TITLE
Retraining from best.pt or last.pt?

### ISSUE THREAD

**Issue by SidneyLann** [ultralytics/ultralytics#16298]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

![image](https://github.com/user-attachments/assets/e581364f-62a1-4219-a09a-205905317213)
nohup yolo train data=/home/sidney/96.yaml model=/home/sidney/runs/detect/train6/weights/best.pt batch=8 epochs=1000 device=0 patience=5000 1>logs/yolo.log 2>&1 &

I should retrain from best.pt or last.pt?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @SidneyLann, thank you for your question and your interest in Ultralytics 🚀! This is an automated response. An Ultralytics engineer will assist you soon.

For guidance on fine-tuning models, please refer to our [Docs](https://docs.ultralytics.com), specifically the sections on [model training tips](https://docs.ultralytics.com/guides/model-training-tips/) which might provide additional insights.

In case you're encountering any issues while retraining, ensure you're using the latest `ultralytics` package. Update your environment with:

```bash
pip install -U ultralytics
```

YOLOv8 runs smoothly in various [environments](https://docs.ultralytics.com/yolov5/environments/), each setup with necessary dependencies. If you prefer real-time chat or discussions, join our [Discord](https://ultralytics.com/discord) 🎧 and [Discourse](https://community.ultralytics.com) respectively, or explore our [Subreddit](https://reddit.com/r/ultralytics).

Remember to check [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml) for operational status. If all is green, it indicates successful operation across multiple platforms.

For further help, please provide more detail if this turns into a 🐛 Bug Report, including a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/).

Happy experimenting! 😊

**Comment by SidneyLann** [comment]
Hi, what's the difference between resuming and restarting?

**Comment by robotaiguy** [comment]
Ehhhh....is it really that simple? Probably...if you're restarting (transfer learning, those weights will be overwritten, so the difference of a hundredth or thousandth of a point of mAP or box loss isn't going to make a difference at the end of the day.

**Comment by SidneyLann** [comment]
What's the standart anwser?

**Comment by glenn-jocher** [comment]
The standard answer depends on your specific training goals. If you're resuming interrupted training, use `last.pt` to continue from where it left off. If you're restarting or fine-tuning, use `
